icekit/navigation/templates/icekit/navigation/navigation_item.html

```diff
@@ -0,0 +1,8 @@
+<li class="ik-nav-item {% if instance in active_navigation_items %}ik-nav-item--active{% endif %}">
+    <a
+        class="ik-nav-item-link {% if instance.html_class %}{{ instance.html_class }}{% endif %}"
```

Comment:
the `{% if %}` is redundant - if there's no class, the `{{ }}` will render empty, no?